src/analysis/inner_product_space/projection.lean

```diff
@@ -772,6 +772,23 @@ lemma orthogonal_projection_mem_subspace_orthogonal_complement_eq_zero
   orthogonal_projection K v = 0 :=
 by { ext, convert eq_orthogonal_projection_of_mem_orthogonal _ _; simp [hv] }
 
+lemma orthogonal_projection_eq_linear_proj [complete_space K] :
+  (orthogonal_projection K : E →ₗ[𝕜] K) =
+  submodule.linear_proj_of_is_compl K _ submodule.is_compl_orthogonal_of_complete_space :=
+begin
+  have : is_compl K Kᗮ := submodule.is_compl_orthogonal_of_complete_space,
+  ext x : 1,
+  nth_rewrite 0 [← submodule.linear_proj_add_linear_proj_of_is_compl_eq_self this x],
+  rw [continuous_linear_map.coe_coe, map_add, orthogonal_projection_mem_subspace_eq_self,
+      orthogonal_projection_mem_subspace_orthogonal_complement_eq_zero (submodule.coe_mem _),
+      add_zero]
+end
+
+lemma orthogonal_projection_eq_linear_proj' [complete_space K] (x : E) :
+  orthogonal_projection K x =
+  submodule.linear_proj_of_is_compl K _ submodule.is_compl_orthogonal_of_complete_space x :=
+by rw [← orthogonal_projection_eq_linear_proj]; refl
```

Comment:
```suggestion
lemma orthogonal_projection_eq_linear_proj [complete_space K] (x : E) :
  orthogonal_projection K x =
  submodule.linear_proj_of_is_compl K _ submodule.is_compl_orthogonal_of_complete_space x :=
linear_map.congr_fun (orthogonal_projection_coe_linear_map_eq_linear_proj K : _) x
```

Reply:
Although on second thoughts, it would make sense to prove these lemmas in the other order to save needing both `congr_fun` and `ext`